packages/web/modals/review-order.tsx

```diff
@@ -210,481 +248,682 @@ export function ReviewOrder({
-                            {formatPretty(amountWithSlippage, {
-                              maxDecimals: 6,
-                            })}{" "}
-                            {quoteType === "out-given-in"
-                              ? toAsset.coinDenom
-                              : fromAsset?.coinDenom}
-                          </span>
-                        )}{" "}
-                      {fiatAmountWithSlippage && (
-                        <span className="text-osmoverse-300">
-                          (~
-                          {formatPretty(fiatAmountWithSlippage, {
-                            ...getPriceExtendedFormatOptions(
-                              fiatAmountWithSlippage.toDec()
-                            ),
-                          })}
-                          )
-                        </span>
-                      )}
-                    </span>
+                    // Do not show skeleton unless there has been no estimation/error yet
+                    !!gasAmount || !!gasError ? (
+                      GasEstimation
+                    ) : (
+                      <Skeleton className="h-5 w-16" />
+                    )
                   }
                 />
-              ) : (
-                <RecapRow
-                  left={t("tradeFee")}
-                  right={
-                    <span className="sm:caption text-bullish-400">
-                      {t("transfer.free")}
+              </div>
+              {isBeyondOppositePrice && orderType === "limit" && (
+                <div className="flex items-start gap-3 rounded-3x4pxlinset border-2 border-solid border-rust-500 p-5">
+                  <Icon
+                    id="alert-triangle"
+                    width={24}
+                    height={24}
+                    className="text-rust-400"
+                  />
+                  <div className="body2 sm:caption flex flex-col gap-1">
+                    <span>
+                      {tab === "buy"
+                        ? t("limitOrders.aboveMarket.title")
+                        : t("limitOrders.belowMarket.title")}
+                    </span>
+                    <span className="text-osmoverse-300">
+                      {tab === "buy"
+                        ? t("limitOrders.aboveMarket.description")
+                        : t("limitOrders.belowMarket.description")}
                     </span>
+                  </div>
+                </div>
+              )}
+              {!is1CTEnabled && (
+                <OneClickTradingPanel
+                  t={t}
+                  transactionParams={transaction1CTParams}
+                  onClick={() =>
+                    setTransaction1CTParams((prev) => {
+                      if (!prev) return;
+
+                      return {
+                        ...prev,
+                        isOneClickEnabled: !prev.isOneClickEnabled,
+                      };
```

Comment:
the only scenario where this can be undefined is when we can't set default value because we haven't got the default param from the local trpc resolver `api.local.oneClickTrading.getParameters`

so we can't set a default value -> if was undefined we want to keep it undefined